client/js/hah-app/team.js

```diff
@@ -28,42 +32,78 @@ class Team extends Component {
 			classes.headshotSize,
 			classes.headshotClicked,
 		)
-
+		const settings = {
+			slidesToShow: 5,
+			infinite: true,
+			centerMode: true,
+			afterChange: idx => {
+				this.setState({ teamMemberId: idx })
+			},
+			swipeToSlide: true,
+			touchMove: true,
+			swipe: true,
+			focusOnSelect: true,
+			draggable: true,
+			responsive: [
+				// {
+				// 	breakpoint: 1300,
+				// 	settings: {
+				// 		slidesToShow: 5,
+				// 		slidesToScroll: 1,
+				// 		infinite: true,
+				// 	},
+				// },
+				{
+					breakpoint: 1300,
```

Comment:
Let's use the standard breakpoints. I just committed [this change to master](https://github.com/HelpAssistHer/help-assist-her/commit/acdc542ae8f8550b10c3464e5ca344391624555f), so if you merge that in, you can actually just use the variable and don't have to hardcode the breakpoint values.

Reply:
Can you also remove the commented out code if it's not going to be used?